goscraper.go

```diff
@@ -72,8 +73,11 @@ func (b *scrapeBuilder) SetUserAgent(s string) ScrapeBuilder {
 	return b
 }
 
-func NewScrapeBuilder() ScrapeBuilder {
-	return &scrapeBuilder{scrapeSettings{userAgent: "GoScraper"}}
+func NewScrapeBuilder(ss ScrapeService) ScrapeBuilder {
+	return &scrapeBuilder{
+		scrapeService:  ss,
```

Comment:
that's a bit puzzling... why do we pass a `ScrapeService` here if it is not used anywhere? we need it only as return for `Build()` 🤔

Reply:
And you are right. 
Removed